### DISCUSSION TITLE
How to make the dialog wider?

### DISCUSSION THREAD

**Discussion by tzatos** [eviden-parallel/community#18] [Q&A] [answered]
I need to make a dialog much wider than its currently possible. How to set the width of the dialog to more than 56rem?

**Comment by Th3S4mur41** [comment, marked as answer]
The dialog style is currently assuming it is mostly used for simple content (single column).
Therefor its width (inline-size) is limited by two factors:
- `--max-line-length`: which is the maximum length suggested for textual content
- `calc(100vw - var(--spacing-xl))`: the viewport size minus some space left and right of the dialog

To get a wider dialog if you e.g. intend to use multiple columns would mean removing the first factor by setting
```css
max-inline-size: calc(100vw - var(--spacing-xl));
```

This is a good indication, that we might want to [reconsider the current style](https://github.com/eviden-parallel/core/issues/118) and add `--max-line-length` to more direct content elements like `<p>` a and `<h1>` instead
